### DISCUSSION TITLE
Volume access / screwed k3s cluster

### DISCUSSION THREAD

**Discussion by wopl** [longhorn/longhorn#3642] [Q&A]
Hi all,
I am running a bare metal k3s cluster with traefik ingress. Unfortunately my cluster screwed up so LonghornUI and Ingress in general are not available. My pods do not attach to volume with

`Unable to attach or mount volumes: unmounted volumes=[myapp], unattached volumes=[kube-api-access-gxkk8 myapp]: timed out waiting for the condition`

PV and PVC are still there and in bound mode. I still can access the damaged cluster with Mirantis Lens and kubectl to operate on it.

Killing the pod (and restart) doesn't help due to cluster problems. I am sure, the Longhorn volume ist still healthy.
I already set up a new hardware/cluster and need to transfer volume data to the new setup.
What could be an approach to transfer the volume (content) to the new machine ? Any chance to overcome the attach problems? Otherwise can I move the complete volume (how?) and get it available on the new hardware?

As quite a few applications are affected, I urgently need some help.
Thanks, Wolfram

**Comment by derekbit** [comment]
One solution is to backup the volume and restore to another cluster.
https://longhorn.io/docs/1.2.3/snapshots-and-backups/backup-and-restore/restore-from-a-backup/

There is a longhorn python client that can help you do backup and restore.
https://longhorn.io/docs/1.2.3/references/longhorn-client-python/

For the attach problem, could you check the longhorn-manager log to see if there is any clue?

**Reply by wopl** [reply]
Thanks, so I found some way as described in 
https://longhorn.io/docs/1.2.3/advanced-resources/data-recovery/export-from-replica/
This will solve my problems.
